patterns/1-initial/extensions-to-manage-contributions-at-scale.md

```diff
@@ -10,14 +10,29 @@ An InnerSource project is receiving too many contributions, making maintenance d
 
 As the number of new feature contributions to a mature InnerSource repository rapidly increases, it adds more burden on code reviews and maintenance. How do we allow for faster release of new features, encouraging innovation and experimentation; while also keeping the repository well maintained?
 
+## Story
+
+There is a strategic project that aims to collect the best innovations within a domain space to one common stack, promoting reuse of a common infrastructure and ensuring a standard user experience. Through InnerSource, various teams in the organization that work within the domain space get an opportunity to collaborate and contribute their innovations to the common codebase, which also offers more opportunities for their work to get adopted. However, a large number of contributions in parallel from several developers is making maintenance of the codebase difficult. This is adding a huge burden on the core team of maintainers who assume ownership over the code quality standards and enable the community through various forms of communication.
+
+Maintainers are burnt out due to:
+
+- Everlasting backlog of pull requests that need to be reviewed.
+- Job Dissatisfaction: Majority of time spent in reviewing code or providing support makes it difficult for maintainers to contribute code themselves or have any time for innovation, making them unhappy.
+- Less feeling of accomplishment: Not all pain results in gain. A lot of time is spent in hardening every contribution for production use through review cycles and giving feedback. Whereas, only a fraction of the new capabilities added solve immediate use cases and gain adoption by users.
+- Time consuming releases: Adding more features to the codebase results in long runnning tests before every release of the project, slowing down the overall process.
+
+A lot of time and investment is going behind releasing a new feature idea to the community of users for exploration.
```

Comment:
I think this is pretty key: The main issue is that there aren't separate paths for "get it to users so they can try it" and "get it to users so that they can build a product around it." I think that's really the main kernel of why extensions are needed! I'm not sure if there's a way to highlight this elsewhere, but it's probably worth bringing to the front somehow.

Reply:
Agreed, and I think it is highlighted in the patlet and problem section , when we say minimal cost and faster release?
` _By offering an extension mechanism outside of the core project, the maintainers enable scaling of project capabilities with minimal cost and maintenance overhead._` 

Story section is meant only for more context into our particular situation. We can think about refining problem statement

Reply:
Yep, that looks great!